### PR TITLE
[Sample/Chat]  Pull Chat SDK version from version made in webpack and not at runtime in the Chat utils

### PR DESCRIPTION
**Pull request by carocao-msft**
# What
[Sample/Chat]  Pull Chat SDK version from version made in webpack and not at runtime in the Chat utils

# Why
Should not be reaching out to external folders to get version information

### CODE REVIEW

File: samples/Chat/src/app/utils/utils.ts [hunk 1]
```diff
@@ -4,13 +4,9 @@
 import { GUID_FOR_INITIAL_TOPIC_NAME } from './constants';
 
 declare let __BUILDTIME__: string; // Injected by webpack
+declare let __CHATVERSION__: string; // Injected by webpack
```
Comment (PorterNan) [inline samples/Chat/src/app/utils/utils.ts:7]: Nice! Didn't know we are injecting this

File: samples/Chat/src/app/utils/utils.ts [hunk 2]
```diff
@@ -4,13 +4,9 @@
...
+export const chatSDKVersion = __CHATVERSION__;
```
Comment (PorterNan) [inline samples/Chat/src/app/utils/utils.ts:8]: Follow the syntax of getBuildTime to make it consistent? Just make it a function and return the value

**Review by JamesBurnside — APPROVED** [review]

File: samples/Meeting/src/app/utils/utils.ts [hunk 3]
```diff
@@ -4,13 +4,9 @@
 import { GUID_FOR_INITIAL_TOPIC_NAME } from './constants';
 
 declare let __BUILDTIME__: string; // Injected by webpack
+declare let __CHATVERSION__: string; // Injected by webpack
+export const chatSDKVersion = __CHATVERSION__;
```
Comment (PorterNan) [inline samples/Meeting/src/app/utils/utils.ts:8]: Same here, follow getBuildTime function style

**Review by PorterNan — APPROVED** [review]

File: samples/Meeting/src/app/utils/utils.ts (hunk 3 above)
Comment (carocao-msft) [reply samples/Meeting/src/app/utils/utils.ts]: Updated :)

**Pull request merged**